src/components/TinderStyleVoting/PostInfoComponents/CardPostHeading.tsx

```
@@ -247,18 +248,51 @@ const CardPostHeading: FC<ICardPostHeadingProps> = (props) => {
+										<BeneficiaryAmoutTooltip
+											key={beneficiary?.address}
+											assetId={beneficiary?.genralIndex || null}
+											requestedAmt={beneficiary?.amount.toString()}
+											className={classNames(className, 'flex')}
+											postId={onchainId ? Number(onchainId) : (onchainId as any)}
+											proposalCreatedAt={created_at as any}
+											timeline={timeline || []}
+											usedInPostPage
+										/>
+									);
+								})}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)